### DISCUSSION TITLE
Booking / Add to cart button does not work when BRW - Booking Rental Plugin WooCommerce is used

### DISCUSSION THREAD

**Discussion by HexaSv** [bootscore/bootscore#569] [Q&A] [answered]
Hi Mate,

I have added BRW - Booking Rental Plugin WooCommerce (a trending rental plugin) to the current bootscore theme (latest update) and it seems that the "Booking" button is not working. I tested with other themes including few default themes that comes with Wordpress.  Any idea how this can be sorted out? or a fix?

**Comment by crftwrk** [comment, marked as answer]
Try to disable ajax cart and let me know if that works https://bootscore.me/documentation/woocommerce/#Disable_AJAX_cart

**Reply by HexaSv** [reply]
Initially, I tried above backend settings without the code on functions.php. With that code on functions.php the issue was solved. Now the add to cart/booking btn works perfectly. Also the rest of the functions in the checkout process works fine. Thank you for the quick solution mate.

**Reply by crftwrk** [reply]
You‘re welcome 👍 Please mark answer as solved.
